src/test/java/com/ably/kafka/connect/MessageSinkMappingTest.java

```diff
@@ -0,0 +1,99 @@
+package com.ably.kafka.connect;
+
+import com.google.gson.JsonObject;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.MessageExtras;
+import io.ably.lib.util.JsonUtils;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.header.Header;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.junit.jupiter.api.Test;
+
+import java.util.Base64;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class MessageSinkMappingTest {
```

Comment:
Have you considered using the
```
// given

// when

// then
```
approach for the tests? I think it would make it easier to understand them if they were structured in this way :wink:

Reply:
Good idea https://github.com/ably/kafka-connect-ably/pull/42/commits/8801cac8db294faab4c01d0804b4305770b1c31d